Alterações para melhorar o Readme. (#6)
tryber/sd-01-week19-recipes-app-3: README.md

- A barra de busca deve aparecer quando a pessoa clicar no ícone de lupa no header. O mesmo serve para escondê-la.

> - A barra de busca deve possuir 3 radio buttons, ingrediente, nome e Primeira letra, eles devem mudar a forma de como será filtrado as receitas as API podem ser vistas [Aqui](https://www.themealdb.com/api.php).Exemplo: Ingredientes selecionado, e busca por chicken.

Reviewer: ```suggestion
- A barra de busca deve possuir 3 _radio buttons_: `Ingrediente`, `Nome` e `Primeira letra`. Eles devem mudar a forma como serão filtradas as receitas.  Os _endpoints_ da API que você deve usar podem ser consultados [aqui para a API de comidas](https://www.themealdb.com/api.php) e [aqui para a API de bebidas](https://www.thecocktaildb.com/api.php). _Exemplo_: Selecionar `Ingrediente` e buscar por `chicken`.
```

   - Após a submissão, 2 tokens devem ser salvos em `localStorage` identificados pelas chaves `meals-token` e `cocktails-token` (o token de teste é sempre "1")

>    - Após a submissão, o e-mail do usuário deve ser salvo em `localStorage` no formato `user: {email: email-do-usuário}` e redirecionado para a tela de listagem de receitas.

Reviewer: ```suggestion
   - Após a submissão, o e-mail do usuário deve ser salvo em `localStorage` no formato `user: {email: email-do-usuário}`.
```

>    - Essa será a tela inicial do projeto, caso o usuário já esteja logado, ele deve ser redirecionado para a tela de listagem de receitas ("Android  - 1" no protótipo )

Reviewer: ```suggestion
   - Após a submissão e validação com sucesso do login, o usuário deve ser redirecionado para a tela de listagem de receitas _("Android  - 1" no protótipo )_.
```

   - A pessoa deve conseguir filtrar por categoria utilizando botões. Cada um com o atributo prefixado: `data-testid=${categoryName}-category-filter`

>    - Ao clicar no filtro de categoria, todas as receitas devem mudar para os dados filtrados da API,caso o filtro seja seleciona de novo, deve retornar as receitas sem nenhum filtro (como se fosse um toggle). Esses campos virão da API que lista categorias `https://www.themealdb.com/api/json/v1/1/list.php?c=list`.

Reviewer: ```suggestion
   - Ao clicar no filtro de categoria, todas as receitas devem mudar para os dados filtrados da API. Caso o filtro seja selecionado de novo, o app deve retornar as receitas sem nenhum filtro _(como se fosse um toggle)_. As categorias disponíveis devem ser obtidas através da API de [comidas](https://www.themealdb.com/api.php) ou [bebidas](https://www.thecocktaildb.com/api.php).
```

>    - Apenas um filtro de categoria deve ser selecionado.

Reviewer: ```suggestion
   - Apenas um filtro de categoria deve poder ser selecionado por vez.
```
